docs/privacyidea.md

```diff
@@ -216,6 +216,41 @@ Use the following example:
         'excludeClientIPs'  => array("10.0.0.0-10.2.0.0", "192.168.178.2"),
     ),
 
+    /**
+     *  This filter is optional, if you want to disable clients by entityID, you should enable it.
+     */
+    22 => array(
+        'class'                => 'privacyidea:checkEntityID',
+
+        /**
+         *  Specify setPath and setKey to equal enabledPath and enabledKey
+         */
+        'setPath'              => 'privacyIDEA',
+        'setKey'               => 'enabled',
+
+        /**
+         *  For a match, the variable $state[$setPath][$setPath] will be set to this value.
+         */
+        'onmatch'              => false,
+
+        /**
+         *  The requesting SAML provider entityID will be tested against this list
+         */
+        'entityids'            => array('http://mySAMLprovider/saml/metadata/'),
+
+        /**
+         *  You may specify additional attribute values as conditions
```

Comment:
...attribute values of the logging in user...

What is the matching? Is it an exact matching or a substring? Less or equal?

So this means, if the user is **not** in this group using, this filter has no effect? Could you please elaborate a bit for a better understanding.

Reply:
it is an exact match.